### ISSUE TITLE
shortcut mapper not accessible to screen readers.

### ISSUE THREAD

**Issue by derekriemer** [notepad-plus-plus/notepad-plus-plus#429]
1.
Open notepad++ with a screen reader such as nvda [nvaccess.org] and go to the settings menu with alt and arrow right to settings.
2. Activate the Shortcut mapper with nvda.
3. Press tab, and note that the list of keys isn't present, the only things present are close and one button called modify.
4. Press the insert key and numpad 7, (or if you are on a laptop keyboard, caps lock + page up) to activate screen review.
If you are on a laptop, press insert+n (numpad insert or extended insert).Now go to preferences and keyboard settings > laptop layout, and also while there, check the use caps-lock as an nvda modifier key check box.Press okay.
when I refer to the nvda key I mean (either one of caps-lock, insert, numpad insert.
Press nvda+numpad 7 or in laptop mode, nvda+pg up. It should say screen review.
5. Now use either numpad 9 to go down by line, or nvda+down arrow to go down by line in laptop mode.
If you do this correctly you should hear a number followed by an action.

These keys should be put into a list (similar to what we have with the windows list so that a screen reader can properly work with them. If you need any tips on how to do this, if you could tell me where the shortcut mapper is in the code, that would help me tell you what to change.

Thanks,
derek

**Comment by dg9ngf** [comment]
There is a lot more nuisance going on with that table. It doesn't surprise me that it's not accessible. See also #909.

**Comment by MetaChuh** [comment]
we are closing this issue due to its age and multiple newer notepad++ versions since it was filed.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html).
if your issue persists, and it is still of relevance to you, please feel free to file a new issue.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
```
[<kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd>](https://notepad-plus-plus.org/community/)
